assets/multiqc_config.yml

```diff
@@ -0,0 +1,11 @@
+report_comment: >
+  This report has been generated by the <a href="https://github.com/nf-core/rnavar" target="_blank">nf-core/rnavar</a>
+  analysis pipeline. For information about how to interpret these results, please see the
+  <a href="https://nf-co.re/rnavar" target="_blank">documentation</a>.
+report_section_order:
+  software_versions:
+    order: -1000
+  "nf-core-rnavar-summary":
+    order: -1001
+
+export_plots: true
```

Comment:
Ahh, I think I have to modify this a bit, to make the order of the tools presented. Otherwise, I see all summaries are in the report.

Reply:
looks good but is still not quite as comprehensive as rnaseq - it might be worth checking this one to see if you can pull over any more config from there. https://github.com/nf-core/rnaseq/blob/master/assets/multiqc_config.yml